dcs/liveries/livery.py

```diff
@@ -104,11 +107,15 @@ def from_lua(code: str, path: str) -> Optional[Livery]:
 
     @staticmethod
     def from_path(path: str) -> Optional[Livery]:
-        if os.path.isdir(path):
-            return Livery.from_directory(path)
-        elif path.endswith(".zip"):
-            return Livery.from_zip(path)
-        return None
+        try:
+            if os.path.isdir(path):
+                return Livery.from_directory(path)
+            elif path.endswith(".zip"):
+                return Livery.from_zip(path)
+            return None
+        except Exception:
```

Comment:
This is really broad. What are you trying to catch? https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/broad-exception-caught.html

Reply:
A similar answer to https://github.com/pydcs/dcs/pull/364#discussion_r1483898175, I unfortunately do not know. However I do believe gaining the knowledge of which livery failed is helpful. It seems like users are downloading liveries of various quality and some aren't set up in the way that this part of the codebase expects.